src/lib/containers/UserCardMedium.tsx

```diff
@@ -7,7 +7,7 @@ import {
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import React from 'react'
 import ReactTooltip from 'react-tooltip'
-import { SynapseClient, SynapseConstants } from '..'
+import { SynapseClient, SynapseConstants } from '../utils'
```

Comment:
This broke the build because it removes the 'SynapseComponents' property...I don't know how we can guard against this using TypeScript or Rollup. One idea I have is to add a CI step to load the UMD build into node to check that the object doesn't get removed. https://sagebionetworks.jira.com/browse/PORTALS-1907

Reply:
The last time this happened, I vaguely recall finding that the way we structure SRC makes this error possible, so a massive restructure may let us avoid this with TypeScript. This would need further exploration, though